src/bsoncxx/oid.cpp

```diff
@@ -23,12 +23,7 @@
 namespace bsoncxx {
 BSONCXX_INLINE_NAMESPACE_BEGIN
 
-const oid::init_tag_t oid::init_tag{};
-
-oid::oid() : _is_valid(false) {
-}
-
-oid::oid(init_tag_t) : _is_valid(true) {
+oid::oid() : _is_valid(true) {
```

Comment:
In this mode, what is the purpose of _is_valid? Can we still have an invalid oid?